docs/source/dev_guide/algorithms/kmeans/includes/computation-batch.rst

```diff
@@ -41,39 +43,58 @@ The K-Means clustering algorithm has the following parameters:
 
 .. list-table::
    :header-rows: 1
+   :widths: 10 10 60
    :align: left
 
    * - Parameter
      - Default Value
      - Description
-   * - algorithmFPType
-     - float
+   * - ``algorithmFPType``
+     - ``float``
      - The floating-point type that the algorithm uses for intermediate computations. Can be float or double.
-   * - method
-     - defaultDense
+   * - ``method``
+     - ``defaultDense``
      - Available computation methods for K-Means clustering:
 
-        -  defaultDense - implementation of Lloyd's algorithm
-        -  lloydCSR - implementation of Lloyd's algorithm for CSR numeric tables
+       For CPU:
 
-   * - nClusters
+        -  ``defaultDense`` - implementation of Lloyd's algorithm
+        -  ``lloydCSR`` - implementation of Lloyd's algorithm for CSR numeric tables
+
+       For GPU:
+
+        -  ``defaultDense`` - implementation of Lloyd's algorithm
+
+   * - ``nClusters``
      - Not applicable
      - The number of clusters. Required to initialize the algorithm.
-   * - maxIterations
+   * - ``maxIterations``
      - Not applicable
      - The number of iterations. Required to initialize the algorithm.
-   * - accuracyThreshold
+   * - ``accuracyThreshold``
      - :math:`0.0`
      - The threshold for termination of the algorithm.
-   * - gamma
+   * - ``gamma``
      - :math:`1.0`
      - The weight to be used in distance calculation for binary categorical features.
-   * - distanceType
-     - euclidean
+   * - ``distanceType``
+     - ``euclidean``
      - The measure of closeness between points (observations) being clustered. The only distance type supported so far is the Euclidian distance.
-   * - assignFlag
-     - true
+   * - **DEPRECATED:** ``assignFlag``
+       
+       **USE INSTEAD:** ``resultsToEvaluate``
+
+     - ``true``
      - A flag that enables computation of assignments, that is, assigning cluster indices to respective observations.
+   * - ``resultsToEvaluate``
+     - ``computeCentroids`` | ``computeAssignments`` | ``computeExactObjectiveFunction``
```

Comment:
Does not look consistent with other algorithms

Reply:
that's the input that was provided to me, and it was explained that it should be written like this

Reply:
OK.  I discussed it offline with Kirill. Let's leave it as it is for now.